src/chart_types/xy_chart/renderer/canvas/xy_chart.tsx

```diff
@@ -168,6 +169,9 @@ class XYChartComponent extends React.Component<XYChartProps> {
           width,
           height,
         }}
+        aria-label="This is a chart"
```

Comment:
Can probably shorten this just to "chart" otherwise we could start every img alt text with "this is..." or something like it.
```suggestion
        aria-label="Chart"
```

Reply:
Thank you! Addressed in 36a71d8